rts/lra/client/src/main/resources/META-INF/beans.xml

```diff
@@ -2,5 +2,5 @@
        xsi:schemaLocation="
          https://jakarta.ee/xml/ns/jakartaee
          https://jakarta.ee/xml/ns/jakartaee/beans_1_2.xsd"
```

Comment:
These are pretty ancient and might be worth upgrading as well (there is no functional change related to that though).

Reply:
I agree with @manovotn. I think "https://jakarta.ee/xml/ns/jakartaee/beans_4_0.xsd" would fit better.